web/hooks/github.go

```diff
@@ -176,8 +178,18 @@ func (wh GitHubWebHook) runAssignmentTests(assignment *pb.Assignment, repo *pb.R
 // recordSubmissionWithoutTests saves a new submission without running any tests
 // for a manually graded assignment.
 func (wh GitHubWebHook) recordSubmissionWithoutTests(data *ci.RunData) {
+	noTestBuildInfo, err := json.Marshal(&ci.BuildInfo{
+		BuildID:   0,
+		BuildDate: time.Now().Format("2006-01-02T15:04:05"),
+		BuildLog:  "No automated tests for this assignment",
+		ExecTime:  1,
+	})
+	if err != nil {
+		wh.logger.Errorf("Error marshalling build info for %s of course %s for student %s", data.Course.Name, data.Assignment.Name, data.JobOwner)
+	}
 	newSubmission := &pb.Submission{
 		AssignmentID: data.Assignment.ID,
+		BuildInfo:    string(noTestBuildInfo),
```

Comment:
(Comment not related to this line, but to Lines 198-200): Suggest returning after the error has been logged and remove the else clause. Further, the log statement on Line 200 should say:
```
Saved manual review submission for user %s for assignment %d.
```

Reply:
Fixed.